neuro_auth_client/security.py

```diff
@@ -36,9 +37,16 @@ def __init__(
     async def identify(self, request: Request) -> Optional[str]:
         auth_header_value = request.headers.get(AUTHORIZATION)
         auth_query_identity = request.query.get(NEURO_AUTH_TOKEN_QUERY_PARAM)
+        ws_subprotocol = request.headers.get(SEC_WEBSOCKET_PROTOCOL)
+        ws_identity = None
+        if ws_subprotocol is not None:
+            for part in ws_subprotocol.strip().split(" "):
+                if part.lower().startswith(WS_BEARER):
+                    ws_identity = part[len(WS_BEARER) :]
+                    break
```

Comment:
minor. this logic seems to be deserving to be a dedicated method.

Reply:
Good idea, done.